.github/workflows/Publish to Azure Artifacts.yml

```diff
@@ -55,6 +56,49 @@ jobs:
       with:
           repository: Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation
           path: Ed-Fi-ODS-Implementation/
+    - name: Verify dependent workflows are success or not 
```

Comment:
This doesn't even get executed when `InitDev ... Package` fails because of the next `if` a few lines above:
```
if: ${{ github.event.workflow_run.conclusion == 'success' || github.event_name == 'workflow_dispatch' }}
```

However,  `Docker Test` continues to get executed because this worklow technically succeeds (an skipped build is not considered a failure).

Could `ahmadnassri/action-workflow-run-wait` Github Action be useful in `Docker Test` and `Publish to Docker Hub` ?

Reply:
github.event.workflow_run.conclusion == 'success'  I removed this condition . Separate step to check success for all 3 workflows before continue the next step